e2e/nx-misc/src/workspace.test.ts

```diff
@@ -58,7 +58,7 @@ describe('Workspace Tests', () => {
       const lib1 = uniq('mylib');
       const lib2 = uniq('mylib');
       const lib3 = uniq('mylib');
-      runCLI(`generate @nx/js:lib ${lib1}/data-access`);
+      runCLI(`generate @nx/js:lib ${lib1}/data-access --unitTestRunner=jest`);
```

Comment:
Why did this have to change?

Reply:
We defaulted to vitest, and some of the file checks are jest specific. Just adding this here since vite setup is covered by other e2e tests already.